torch/testing/_internal/jit_metaprogramming_utils.py

```diff
@@ -286,7 +288,7 @@ def script_fn(*args, **kwargs):
         fn, tensors = gen_script_fn_and_args(method_name, func_type, *args, **kwargs)
         self.assertExportImport(fn.graph, tensors)
         output = output_process_fn(fn(*tensors))
-        script_fn.last_graph = fn.graph_for(*tensors)
+        setattr(script_fn, 'last_graph', fn.graph_for(*tensors))
```

Comment:
I dont think this way of setting function attribute is consider safe under mypy rules. this https://github.com/python/mypy/issues/2087 might be useful